src/pages/LoanFinding/LoanFinding.vue

```diff
@@ -299,6 +311,16 @@ export default {
 		}
 
 		this.initializeFiveDollarsNotes();
+
+		// Relending test for users with balance
+		const { version } = trackExperimentVersion(
```

Comment:
Then with the change described in the other comment, we can check the balance before calling `trackExperimentVersion`